modules/vm/iam.tf

```diff
@@ -15,6 +15,24 @@ resource "aws_iam_role_policy_attachment" "systems-manager-policy" {
   policy_arn = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
 }
 
+resource "aws_iam_role_policy" "instance_ssm" {
+  name   = "${var.resource_name_prefix}-graphdb-ssm-describe"
+  role   = var.iam_role_id
+  policy = data.aws_iam_policy_document.instance_ssm.json
+}
+
+data "aws_iam_policy_document" "instance_ssm" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "ssm:DescribeParameters"
+    ]
+
+    resources = ["*"]
```

Comment:
I wonder if we could limit the scope of this policy, e.g. `/${var.resource_name_prefix}/graphdb/*`

Reply:
doesn't seem like it. 
```
 Error: putting IAM Role (vik-graphdb-20240322052336311700000001) Policy (vik-graphdb-ssm-describe): MalformedPolicyDocument: Resource /vik/graphdb/* must be in ARN format or "*".
│       status code: 400, request id: b87e1db6-27fd-4b97-a61f-53e8e0a4e2eb
│
```
Requires ARN format and it would be different for every deployment and regions for example the ARN for admin_password is  `arn:aws:ssm:eu-central-1:777777777777:parameter/gdbtest/graphdb/admin_password`

Reply:
I think it can be parse dynamically.

Reply:
What about 
`arn:aws:ssm:eu-central-1:777777777777:parameter/${var.resource_name_prefix}/graphdb/*` ?

Reply:
Reduced to arn:aws:ssm:eu-central-1:777777777777:*